os/hal/ports/KINETIS/K20x/hal_lld.h

```diff
@@ -22,8 +22,8 @@
  * @{
  */
 
-#ifndef _HAL_LLD_H_
-#define _HAL_LLD_H_
+#ifndef HAL_LLD_H_
+#define HAL_LLD_H_

```

Comment:
IMO it would be better if you left out the _ at the end. But I don't mind very much!